packages/lib/passport-interface/src/genericIssuanceTypes.ts

```diff
@@ -59,6 +59,36 @@ export type BasePipelineDefinition = z.infer<
   typeof BasePipelineDefinitionSchema
 >;
 
+/**
+ * Pipeline definitions can also include manually-added tickets. Pipelines that
+ * support this will create tickets according to these specifications, in
+ * addition to those loaded from their primary data source.
+ */
+const ManualTicketSchema = z.object({
+  /**
+   * The ID of the ticket.
+   */
+  id: z.string().uuid(),
+  /**
+   * The generic issuance UUID of the event that the ticket is for.
+   */
+  eventId: z.string().uuid(),
+  /**
+   * The generic issuance UUID for the product/ticket type.
+   */
+  productId: z.string().uuid(),
+  /**
+   * The email to assign the ticket to.
+   */
+  attendeeEmail: z.string().email(),
+  /**
+   * The full name of the attendee.
+   */
+  attendeeName: z.string().min(1)
```

Comment:
@rrrliu - should manual tickets have any other fields?

Reply:
the only missing thing here is the consumed state, but I assume that will be covered in #1544